Userland/DevTools/Profiler/Profile.cpp

```diff
@@ -334,7 +334,7 @@ ErrorOr<NonnullOwnPtr<Profile>> Profile::load_from_perfcore_file(StringView path
                 .executable = executable,
             };
 
-            auto sampled_process = adopt_own(*new Process {
+            auto sampled_process = adopt_nonnull_own_or_enomem(new (nothrow) Process {
```

Comment:
I _think_ `try_make` doesn't work with the `.pid` (etc) syntax, but I haven't actually tried it.